jenkins/ci.suse.de/ccp.yaml

```diff
@@ -0,0 +1,66 @@
+- job:
+    name: 'ccp-full-periodic'
+    node: cloud-trackupstream
+
+    parameters:
+      - string:
+          name: ccp_repo
+          default: https://github.com/SUSE-Cloud/socok8s.git
+          description: The git repository to use
+      - string:
+          name: PREFIX
```

Comment:
This looks like a parameter you use to differentiate between artifacts created by different job builds. If you're also considering running multiple instances of this job in parallel, you need to take the following into account:
 - enable concurrency for the job (`concurrent: true` in the JJB job definition)
 - job instances will use the same Jenkins workspace path and trample over each-other, unless you explicitly configure one that is unique for every build (e.g. `workspace: $JOB_NAME-$BUILD_NUMBER`). 
 - if you enable unique workspaces, you should also consider cleaning them up, at the end of the build, e.g:

```
    publishers:
      - workspace-cleanup:
          clean-if:
            - failure: false
            - aborted: false
            - unstable: false
```

, instead of:

```
    wrappers:
       - workspace-cleanup:
           dirmatch: true
```

, which is cleaning up the workspace before the job is executed

Reply:
PREFIX is not used to differ between different builds itself, but something is based on PREFIX that will make use of it.
I am not considering using multiple instances of this job in parallel. In fact, I don't want those in parallel.
Just one is good enough (it will run as periodic, so no risk of overlap anyway). Reusing the environment is part of the deal.